src/Runtime/Optimized/Physics/mmdBulletPhysics.ts

```
@@ -0,0 +1,640 @@
+                },
+                limits,
+                scene
+            );
+
+            bodyA.addConstraint(bodyB, constraint);
+
+            constraints[i] = constraint;
+
+            // adjust the physics mode of the rigid bodies
+            // ref: https://web.archive.org/web/20140815111315/www20.atpages.jp/katwat/wp/?p=4135
+            const nodeA = nodes[joint.rigidbodyIndexA]!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)